### DISCUSSION TITLE
Synchronizing calendar gets stuck

### DISCUSSION THREAD

**Discussion by glen-84** [kewisch/gdata-provider#816] [Support]
![image](https://github.com/user-attachments/assets/c41ede1e-36ef-403d-b236-8d99f6f1afc8)

It just stays like this. Sometimes there are multiple entries in the activity manager for the same thing.

Is there some way to debug this? Any logs?

128.3.1esr (64-bit)

**Comment by Edobeta** [comment]
Same issue here
![Immagine 2024-10-11 235359](https://github.com/user-attachments/assets/2a892d85-2a64-404f-9ef4-72ad6cc0c256)

**Comment by kewisch** [comment]
You can enable logging here: https://github.com/kewisch/gdata-provider/wiki#enabling-debugging

**Comment by QuinceTart10** [comment]
I'm having the same issue on my machine and there doesn't seem to be anything out of the ordinary in the logs. It seems like there's something up with the regional calendars though

**Comment by DenMar51** [comment]
Same issue here with Holidays in Canada.
![Holidays in Canada](https://github.com/user-attachments/assets/d34ed011-eef0-42a6-a5b7-ec974d2e7da8)
